h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -5,32 +5,50 @@
 import water.api.API;
 import water.api.schemas3.KeyV3;
 import water.api.schemas3.ModelParametersSchemaV3;
+import hex.Model;
+
 
 
 public class StackedEnsembleV99 extends ModelBuilderSchema<StackedEnsemble,StackedEnsembleV99,StackedEnsembleV99.StackedEnsembleParametersV99> {
   public static final class StackedEnsembleParametersV99 extends ModelParametersSchemaV3<StackedEnsembleModel.StackedEnsembleParameters, StackedEnsembleParametersV99> {
-    static public String[] fields = new String[]{
+    static public String[] fields = new String[] {
       "model_id",
       "training_frame",
       "response_column",
       "validation_frame",
       "base_models",
+      "metalearner_nfolds",
+      "metalearner_fold_assignment",
+      //"metalearner_fold_column",
       "keep_levelone_frame",
-      //"selection_strategy",
     };
 
-    /*
-    @API(help = "Strategy for choosing which models to stack.", values = { "choose_all" }, gridable = false)
-    public StackedEnsembleModel.StackedEnsembleParameters.SelectionStrategy selection_strategy;
-
-    @API(help = "List of model ids which we can stack together.  Which ones are chosen depends on the selection_strategy (currently, all models will be used since selection_strategy can only be set to choose_all).  Models must have been cross-validated using nfolds > 1, fold_assignment equal to Modulo, and keep_cross_validation_folds must be set to True.", required = true)
-    public KeyV3.ModelKeyV3 base_models[];
-    */
 
     @API(help = "List of model ids which we can stack together. Models must have been cross-validated using nfolds > 1, and folds must be identical across models.", required = true)
     public KeyV3.ModelKeyV3 base_models[];
 
     @API(help = "Keep level one frame used for metalearner training.")
     public boolean keep_levelone_frame;
+
+    @API(level = API.Level.critical, direction = API.Direction.INOUT, gridable = true,
+            help = "Number of folds for K-fold cross-validation of the metalearner algorithm (0 to disable or >= 2).")
+    public int metalearner_nfolds;
+
+    // TO DO: Add metalearner_fold_column
```

Comment:
Might be good to tag the JIRA ticket here.

Reply:
Ok sure, will do.